start.sh

```diff
@@ -40,6 +40,8 @@ function init_conda() {
     echo "no existing environment found"
     conda create --use-index-cache --clone root -n rstudio --copy -y \
     && chown -R "${USER}:${GROUP}" "/home/$USER/.conda/"
+    chmod -R 775 "/home/$USER/.conda/envs/rstudio/bin" \
+    && chmod -R 775 "/home/$USER/.conda/envs/rstudio/lib/R/bin"
```

Comment:
So the permissions will be adjusted only just after the creation of the `rstudio` conda environment (it wouldn't make sense to run this at each start-up anyway I guess).

Good to keep in mind, in case we see weird permission errors we'll have to reset their environment and trigger a clean recreation of the conda environment 👌

Reply:
Yes this one is a little strange, but it seems to be due to the later version of conda.
It seems to work for now (and is ok after we install new packages via conda) but we might need to revisit it in the future.